google/cloud/spanner/admin/integration_tests/instance_admin_integration_test.cc

```diff
@@ -73,16 +73,23 @@ bool Emulator() {
 class CleanupStaleInstances : public ::testing::Environment {
  public:
   void SetUp() override {
-    EXPECT_STATUS_OK(
-        spanner_testing::CleanupStaleInstances(Project(ProjectId())));
+    spanner_admin::InstanceAdminClient instance_client(
```

Comment:
We named these `instance_admin_client`, etc., in instance_admin_rest_integration_test.cc.  The fewer differences in the duplication the better I reckon.  That is, I'd like to be able to diff them and just see `s/Rest//`.